src/test/java/org/cactoos/func/FuncOfTest.java

```diff
@@ -50,30 +50,6 @@ public void convertsProcIntoFunc() throws Exception {
         );
     }
 
-    @Test
-    public void convertsProcWithNoResultIntoFunc() throws Exception {
```

Comment:
@SharplEr leave this test for now until `FuncOf` is refactored

Reply:
@Vatavuk Done.